cnf-tests/entrypoint/test-run.sh

```diff
@@ -3,19 +3,19 @@ set -e
 # Setting -e is fine as we want both config and validiation to succeed
 # before running the "real" tests.
 
-TEST_SUITES=${TEST_SUITES:-"validationsuite configsuite cnftests"}
+LATENCY_TEST_RUN="${LATENCY_TEST_RUN:true}"
```

Comment:
Did you try to set a default value here?
Because it should be `"${LATENCY_TEST_RUN:=true}"` note the `=`

Reply:
Not trying to assign a default value but to get the assigned value or the default (true here) if variable is not defined.
In any case it was wrong.
Thanks for pointing the problem. Nice catch